src/ray/protobuf/core_worker.proto

```diff
@@ -87,16 +87,6 @@ message DirectActorCallArgWaitCompleteRequest {
 message DirectActorCallArgWaitCompleteReply {
 }
 
-message WorkerLeaseGrantedRequest {
-  // Address of the leased worker.
-  string address = 1;
-  // Port of the leased worker.
-  int32 port = 2;
-}
-
-message WorkerLeaseGrantedReply {
-}
-
```

Comment:
nice. Good to use request & reply instead of separate set of messages for request and grant.